Sources/FluentBenchmark/Tests/MigratorTests.swift

```diff
@@ -53,6 +54,80 @@ extension FluentBenchmarker {
             try migrator.revertAllBatches().wait()
         }
     }
+
+    private func testMigrator_sequence() throws {
+        try self.runTest(#function, []) {
+
```

Comment:
This could have something like:
```swift
guard self.databases.ids.count >= 2 else {
    XCTFail("need at least two dbs for this test")
}
let db1 = self.databases.ids[0]
let db2 = self.databases.ids[1]

...
```